packages/storybook/src/nimble/select/select-matrix.stories.ts

```diff
@@ -100,6 +102,29 @@ export const blankListOption: StoryFn = createStory(
     </${selectTag}>`
 );
 
+const playFunction = async (): Promise<void> => {
+    await Promise.all(
+        Array.from(document.querySelectorAll<Select>('nimble-select')).map(
+            async select => {
+                const arrowDownEvent = new KeyboardEvent('keydown', {
+                    key: keyArrowDown
+                });
+                select.dispatchEvent(arrowDownEvent);
+                await waitForUpdatesAsync();
+            }
+        )
+    );
+};
+
+export const navigateToDifferentOption: StoryFn = createStory(
+    html`<${selectTag} open>
```

Comment:
@atmgrifter00 fyi in #2073 we started setting explicit widths on select / combobox stories

```suggestion
    html`<${selectTag} open style="width: 250px;">
```

Reply:
Gah! I'll have to address this in a follow-up PR. Sorry.

Reply:
I did it in this PR by merging the above comment, you're all good, was just an FYI for future story work